.circleci/config.yml

```diff
@@ -120,31 +153,159 @@ jobs:
+            curl -sSL --retry 5 --connect-timeout 15 "$DATA_NIPYPE_FSL_COURSE" | tar xz
+
+            export DATA_NIPYPE_FSL_FEEDS="${OSF_NIPYPE_URL}/57f473066c613b01f113e7af"
+            curl -sSL --retry 5 --connect-timeout 15 "$DATA_NIPYPE_FSL_FEEDS" | tar xz
+      - run:
+          name: Run tests
+          no_output_timeout: 2h
+          environment:
+            WORKDIR: /home/circleci/work
+          command: |
+            mkdir -p "$WORKDIR"
+            chmod -R 777 "$WORKDIR"
+            bash /home/circleci/nipype/.circleci/test_py3_fmri_fsl_feeds_linear_l1.sh
+            bash /home/circleci/nipype/.circleci/test_py3_fmri_fsl_reuse_linear_l1.sh
+            bash /home/circleci/nipype/.circleci/test_py3_fmri_spm_linear_3d.sh
+            bash /home/circleci/nipype/.circleci/test_py3_fmri_spm_linear_4d.sh
+      - store_artifacts:
+          path: /home/circleci/work/tests
 
+  test_py3_fmri_spm_dartel_linear:
+    docker:
+      - image: docker:17.10.0-ce-git
+    working_directory: /home/circleci/nipype
+    steps:
+      - setup_remote_docker
+      - attach_workspace:
+          at: /tmp
+      - run:
+          name: Load saved Docker images.
+          no_output_timeout: 60m
+          command: |
+            docker load < /tmp/docker/nipype-base-latest-py36-py27.tar.gz
+      - checkout:
+          path: /home/circleci/nipype
+      - run:
+          name: Get codecov
+          command: |
+            pip install --no-cache-dir codecov
+      - run:
+          name: Download test data
+          no_output_timeout: 20m
+          working_directory: /home/circleci/examples
+          environment:
+            OSF_NIPYPE_URL: "https://files.osf.io/v1/resources/nefdp/providers/osfstorage"
+          command: |
+            export DATA_NIPYPE_TUTORIAL_URL="${OSF_NIPYPE_URL}/57f4739cb83f6901ed94bf21"
+            curl -sSL --retry 5 --connect-timeout 15 "$DATA_NIPYPE_TUTORIAL_URL" | tar xj
+
+            export DATA_NIPYPE_FSL_COURSE="${OSF_NIPYPE_URL}/57f472cf9ad5a101f977ecfe"
+            curl -sSL --retry 5 --connect-timeout 15 "$DATA_NIPYPE_FSL_COURSE" | tar xz
+
+            export DATA_NIPYPE_FSL_FEEDS="${OSF_NIPYPE_URL}/57f473066c613b01f113e7af"
+            curl -sSL --retry 5 --connect-timeout 15 "$DATA_NIPYPE_FSL_FEEDS" | tar xz
+      - run:
+          name: Run tests
+          no_output_timeout: 2h
+          environment:
+            WORKDIR: /home/circleci/work
+          command: |
+            mkdir -p "$WORKDIR"
+            chmod -R 777 "$WORKDIR"
+            bash /home/circleci/nipype/.circleci/test_py3_fmri_spm_dartel_linear_l1.sh
+            bash /home/circleci/nipype/.circleci/test_py3_fmri_spm_dartel_linear_l2.sh
+      - store_artifacts:
+          path: /home/circleci/work/tests
+
+  test_fmri_spm_nested_multiproc:
+    docker:
+      - image: docker:17.10.0-ce-git
+    working_directory: /home/circleci/nipype
+    steps:
+      - setup_remote_docker
```

Comment:
the remote docker might be useful when building, but it is not necessary for running. I may be wrong, but I think that remote docker does not allow mounting, and you need to do `docker cp`. The different jobs may be just using the local docker install. IMHO this line can be removed on the test jobs.